TuneUp/TuneUpViewExtension.cs

```diff
@@ -54,7 +54,11 @@ public override void Loaded(ViewLoadedParams p)
 
             // Add this view extension's menu item to the Extensions tab or View tab accordingly.
             var dynamoMenuItems = p.dynamoMenu.Items.OfType<MenuItem>();
-            var extensionsMenuItem = dynamoMenuItems.Where(item => item.Header.ToString() == "_Extensions");
+            // TODO: Investigate why TuneUpMenuItem is not added to the specified MenuItem
+            // When _Extensions is specified, TuneUpMenuItem goes to the View menu in Dynamo 3.0-
+            // and does not appear in Dynamo 3.1+.
+            // We need to specify _View for TuneUpMenuItem to be added to the Extensions menu
+            var extensionsMenuItem = dynamoMenuItems.Where(item => item.Header.ToString() == "_View");
```

Comment:
so after this change it is showing up under extensions menu? that is wierd.

Reply:
Indeed. Looking and the old master seem like this hasn't worked before either. In old demos Show TuneUp is always called from the View menu although the code was specifying _Extensions.